cmds/core/init/init_linux.go

```diff
@@ -6,17 +6,57 @@ package main
 
 import (
 	"log"
+	"os"
+	"path/filepath"
 	"strconv"
+	"strings"
 	"syscall"
 
 	"github.com/u-root/u-root/pkg/cmdline"
+	"github.com/u-root/u-root/pkg/kmodule"
 )
 
+// installModules installs kernel modules (.ko files) from /lib/modules.
+// Useful for modules that need to be loaded for boot (ie a network
+// driver needed for netboot)
+func installModules() {
```

Comment:
I think this should be shared with whatever the modprobe command already does. At least it needs to be a library function in kmodule

cc @rminnich